sdk/servicebus/azure-messaging-servicebus/src/main/java/module-info.java

```diff
@@ -3,6 +3,7 @@
 
 module com.azure.messaging.servicebus {
     requires transitive com.azure.core.amqp;
+    requires transitive com.azure.core;
```

Comment:
since core-amqp transitively depend on `azure-core` , this should not be needed.

Reply:
According to your comment, it has been corrected in the new version